sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -64,6 +64,11 @@ def __init__(self, endpoint, credential, **kwargs):
             }
         )
 
+        try:
+            endpoint = endpoint.rstrip("/")
+        except AttributeError:
+            raise ValueError("Parameter 'endpoint' must be a string.")
+
```

Comment:
it didn't really matter before since we weren't manipulating the endpoint, but just realized that now that we're rstriping it we should make sure `self._endpoint` on L26 also gets this value. It's used when any `get_*client` call is made.